src/test/resources/com/conveyal/datatools/mtc-rtd-mock-responses/mtc-rtd-mock-responses.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't be included.

Reply:
Thanks, removed in https://github.com/ibi-group/datatools-server/pull/438/commits/3fc8d79507c45e0debea70254c05cc34c4686535.